activerecord/test/cases/migration/compatibility_test.rb

```diff
@@ -558,7 +558,7 @@ def migrate(x)
           ActiveRecord::Migrator.new(:up, [migration], @schema_migration).migrate
 
           testings_id_column = connection.columns(:more_testings).find { |el| el.name == "testings_id" }
-          assert_equal "integer", testings_id_column.sql_type
```

Comment:
I think using `assert_match` would be more resilient - the `downcase` might be removed by an unsuspecting future developer, e.g:

``` ruby
assert_match /integer/i, testings_id_column.sql_type
```

Reply:
Absolutely